src/RepoDriver.sol

```diff
@@ -161,17 +165,43 @@ contract RepoDriver is ERC677ReceiverInterface, ERC2771Context, Managed {
         userId = (userId << 216) | nameEncoded;
     }
 
+    /// @notice Initializes the AnyApi operator configuration.
+    /// Callable only once, and only before any calls to `updateAnyApiOperator`.
+    /// @param operator The initial address of the AnyApi operator.
+    /// @param jobId The initial AnyApi job ID used for requesting user owner updates.
+    /// @param defaultFee The initial fee in Link for each user owner.
+    /// update request when the driver is covering the cost.
+    function initializeAnyApiOperator(OperatorInterface operator, bytes32 jobId, uint96 defaultFee)
+        public
+        whenNotPaused
+    {
+        require(!_repoDriverAnyApiStorage().isInitialized, "Already initialized");
+        _updateAnyApiOperator(operator, jobId, defaultFee);
+    }
+
     /// @notice Updates the AnyApi operator configuration. Callable only by the admin.
     /// @param operator The new address of the AnyApi operator.
     /// @param jobId The new AnyApi job ID used for requesting user owner updates.
-    /// @param defaultFee The new fee in Link for each user owner
+    /// @param defaultFee The new fee in Link for each user owner.
     /// update request when the driver is covering the cost.
     function updateAnyApiOperator(OperatorInterface operator, bytes32 jobId, uint96 defaultFee)
         public
         whenNotPaused
         onlyAdmin
+    {
+        _updateAnyApiOperator(operator, jobId, defaultFee);
+    }
+
+    /// @notice Updates the AnyApi operator configuration. Callable only by the admin.
+    /// @param operator The new address of the AnyApi operator.
+    /// @param jobId The new AnyApi job ID used for requesting user owner updates.
+    /// @param defaultFee The new fee in Link for each user owner.
+    /// update request when the driver is covering the cost.
+    function _updateAnyApiOperator(OperatorInterface operator, bytes32 jobId, uint96 defaultFee)
+        internal
     {
         RepoDriverAnyApiStorage storage storageRef = _repoDriverAnyApiStorage();
+        storageRef.isInitialized = true;
```

Comment:
It would be cheaper for a regular update call if this is done in the `initializeAnyApiOperator`. Since the boolean slot is shared with the `nonce` which is not updated here. 

A bit more expensive and not so clean for the init. Reading the slot twice.